requirements-release_test.txt

```diff
@@ -7,5 +7,6 @@ pytest-xdist
 numpy==2.0.0; python_version >= "3.10" and python_version <= "3.12"
 numpy==2.2.0; python_version>="3.13"
 protobuf==6.31.1
+version_utils
```

Comment:
This is not right. version_utils is not a package we download (an internal module)